pwa.php

```diff
@@ -17,7 +17,7 @@
  * License:           GPL-2.0+
  * License URI:       http://www.gnu.org/licenses/gpl-2.0.txt
  * GitHub Plugin URI: https://github.com/xwp/pwa-wp
- * Requires PHP:      5.2
+ * Requires PHP:      5.6
  * Requires WP:       4.9
```

Comment:
If we require PHP 5.6 because of WordPress core doing so, we should also require WordPress 5.2 here.